src/org/intellij/erlang/debugger/xdebug/ErlangXDebugProcess.java

```diff
@@ -358,6 +361,25 @@ private OSProcessHandler runDebugTarget() throws ExecutionException {
     return erlangProcessHandler;
   }
 
+  private void setupErlangAppConfig(GeneralCommandLine commandLine) throws ExecutionException {
+    ErlangRunConfigurationBase<?> runConfig = getRunConfiguration();
+    if (runConfig.getDebugOptions().isLoadingConfig()) {
+      commandLine.addParameters("-config", runConfig.getDebugOptions().getAppConfig());
+    }
+  }
+
+  private void setupErlangRebarDependencies(GeneralCommandLine commandLine) throws ExecutionException {
+    ErlangRunConfigurationBase<?> runConfig = getRunConfiguration();
+    if (runConfig.getDebugOptions().isIncludingRebarDependencies()) {
+      Set<String> dependencies = RebarConfigUtil.getRebarDependencies(
+              runConfig.getConfigurationModule().getModule(),
+              runConfig.getDebugOptions().isFetchingDependencies());
+
+      for (String dep : dependencies)
+        commandLine.addParameters("-pa", dep);
+    }
```

Comment:
Rebar dependencies are supposed to be set when you import a rebar project. 

Also, getting dependencies like that implies that you've built them with rebar before running the run configuration.

I suggest we split this request in two: first - setting -config option in run configurations, and second - re-importing project to update rebar dependencies.